src/components/form/examples/dynamic-form.tsx

```diff
@@ -44,28 +44,39 @@ export class DynamicFormExample {
     }
 
     public render() {
-        return [
-            <textarea onChange={this.handleTextChange}>{this.text}</textarea>,
-            <br />,
-            <limel-form
-                onChange={this.handleFormChange}
-                onValidate={this.handleValidate}
-                value={this.formData}
-                schema={this.schema}
-            />,
-            <limel-example-value value={this.formData} />,
-            <limel-example-value label="Errors" value={this.errors} />,
-        ];
+        return (
+            <Host>
+                <limel-code-editor
+                    language="json"
+                    lineNumbers={true}
+                    lint={true}
+                    fold={true}
+                    onChange={this.handleTextChange}
+                    value={this.text}
+                />
+                ,
+                <br />,
+                <limel-form
+                    onChange={this.handleFormChange}
+                    onValidate={this.handleValidate}
+                    value={this.formData}
+                    schema={this.schema}
+                />
+                ,
+                <limel-example-value value={this.formData} />,
+                <limel-example-value label="Errors" value={this.errors} />,
```

Comment:
```suggestion
                <limel-code-editor
                    language="json"
                    lineNumbers={true}
                    lint={true}
                    fold={true}
                    onChange={this.handleTextChange}
                    value={this.text}
                />
                <br />
                <limel-form
                    onChange={this.handleFormChange}
                    onValidate={this.handleValidate}
                    value={this.formData}
                    schema={this.schema}
                />
                <limel-example-value value={this.formData} />
                <limel-example-value label="Errors" value={this.errors} />
```

The commas aren't needed anymore, and if kept, they'll render in the DOM as `,`s